airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -77,14 +96,61 @@ definitions:
         type: string
         enum: [ApiKeyAuthenticator]
       api_token:
+        title: API Key
+        description: The API key to inject in the request
         type: string
+        interpolation_context:
+          - config
+        examples:
+          - "{{ config['api_key'] }}"
+          - "Token token={{ config['api_key'] }}"
       header:
+        title: Header
+        description: The name of the header that will be injected in the request
         type: string
+        interpolation_context:
+          - config
+        examples:
+          - Authorization
+          - Api-Token
+          - X-Auth-Token
       $parameters:
         type: object
         additionalProperties: true
+  AuthFlow:
+    title: "Auth flow"
+    description: |-
```

Comment:
I didn't update the description for the `Auth flow`. It's very long, but I think it's a very complicated component. An alternative would be to link to its analogous description in the airbyte-protocol